openmrs-watcher/src/main/resources/camel/db-event-processor.xml

```diff
@@ -7,12 +7,9 @@
 
         <process ref="id-setting-event-processor" />
 
-        <when>
-            <simple>${exchangeProperty.db-event-destinations} == null</simple>
-            <setProperty name="db-event-destinations">
-                <simple>{{db-event.destinations}}</simple>
-            </setProperty>
-        </when>
+        <setProperty name="db-event-destinations">
```

Comment:
This is not enough to work with multiple destinations, there are other underlying issues that stay unresolved.

Reply:
The {{watcher-retry-item-handle}} is the only route in the codebase that sets the value for the {{db-event-destinations}} exchange property, this is because a retry item is meant to be redelivered ONLY to the route it belongs to, so this can never be null for a retry item execution, but it will always be null when processing a db event from the {{debezium_event_queue}} table meaning it will be delivered to all routes if we supported multiple. Like I said on the ticket, this is just one of the areas where the design is flawed, but there are others which is why we can't support multiple destinations for now.